components/cam/src/physics/cam/shoc.F90

```diff
@@ -2190,9 +2191,10 @@ subroutine shoc_assumed_pdf_vv_parameters(&
   real(rtype) :: sqrtw2t
 
   ! parameters
-  real(rtype), parameter :: w_tol_sqd=(2.e-2_rtype)**2
+  real(rtype) :: w_tol_sqd
+  w_tol_sqd = bfb_square(2.e-2_rtype)
 
-  Skew_w=w3var/w_sec**(3./2.)
+  Skew_w=w3var/bfb_pow(w_sec,(3./2.))
```

Comment:
Should `3./2.` be changed to `3._rtype/2._rtype`?

Reply:
Better: `bfb_sqrt(bfb_cube(w_sec))`.